### ISSUE TITLE
[SliderItem] Accessibility improvements

### ISSUE THREAD

**Issue by perruche** [studiometa/ui#136]
## Issues
1. The `SliderItem` add `aria-hidden="true/false"` depending on if he is visible or not. This is a bad practice for accessibility, this makes any focusable element inside a slide that has `aria-hidden="true"` innaccessible with assistive technology.
2. Each SliderItem should have an accessible name using aria-label or aria-labelledby

## Suggested fix
1. Remove the aria-hidden
2. Update documentation to include best practice in the exemples

Documentation for the Carousel pattern to implement is available [here](https://www.w3.org/WAI/ARIA/apg/patterns/carousel/)
